tests/gocase/unit/quit_test.go

```diff
@@ -0,0 +1,101 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package unit
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+
+	"github.com/apache/incubator-kvrocks/tests/gocase/util"
+	"github.com/stretchr/testify/require"
+)
+
+func formatCommand(args []string) string {
+	str := fmt.Sprintf("*%d\r\n", len(args))
+	for _, arg := range args {
+		str += fmt.Sprintf("$%d\r\n%s\r\n", len(arg), arg)
+	}
+	return str
+}
+
+func TestQuit(t *testing.T) {
+	srv := util.StartServer(t, map[string]string{})
+	defer srv.Close()
+	rdb := srv.NewTCPClient()
+	defer func() { require.NoError(t, rdb.Close()) }()
```

Comment:
```suggestion
```
I think u can move it into every t.Run.

Reply:
> Hi, the first `rdb := srv.NewTCPClient()` is not used, I think u can move it into every `t.Run` .Thanks for your great contribution.

I think that the reconnect is not reflected after these changes.